### PR TITLE
Run some maintainers with higher interval

### PR DESCRIPTION
**Pull request by hmusum**
When deployments exist on all config servers each server will deploy,
so do it with 3 times higher interval

### CODE REVIEW

**Review by hakonhall — APPROVED** [review]

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java:183]: `PeriodicApplicationMaintainer` runs every minute (`redeployMaintainerInterval`), this is the `minTimeBetweenRedeployments`, so I don't think we should change this.

**Review by freva — COMMENTED** [review]
In addition, I suggest scaling 3x `operatorChangeRedeployInterval`, `retiredInterval`, `autoscalingInterval` (going by usage of `Deployer`).

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java
Comment (hmusum) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java:183]: Ouch, yes, thanks

File: node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java
Comment (freva) [inline node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java:189]: This can also be reduced back now, it does not deploy (assume it was increased because of periodic interval change?)

Comment (hmusum) [reply node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java]: Yes, thanks, fixed

**Review by freva — APPROVED** [review]
LGTM, not sure if you saw my suggestion for other maintainers that could be have their interval increased.

**Comment by hmusum** [comment]
No, didn't see it, will have a look

**Review by freva — APPROVED** [review]
:100:

**Pull request merged**
